server/build.gradle

```diff
@@ -79,12 +79,12 @@ jacocoTestReport {
   }
 
   afterEvaluate {
-    // This excludes the `Server` class from the coverage report. We don't
-    // have any good way to test the `Server` class directory (we'd have
+    // This excludes the `Main` class from the coverage report. We don't
+    // have any good way to test the `Main` class (we'd have
     // to somehow fake incoming HTTP requests), so we are just
     // leaving it out of the coverage report and the coverage limits.
     classDirectories.setFrom(files(classDirectories.files.collect {
-      fileTree(dir: it, exclude: 'umm3601/Server.class')
+      fileTree(dir: it, exclude: ['umm3601/Main.class', 'umm3601/Server.class'])
```

Comment:
The words say we will exclude Main, but we are also excluding Server. Is that intentional? If so, should both be used/listed in the words?

Reply:
I had to keep `Server` excluded because without the PRs that added all the `Server` tests, the code coverage wasn't high enough to pass. Now that you've approved those PRs, let me look at the code coverage and see what happens.

If it doesn't get us there, my inclination is to merge this in since it changes so much, and then create an issue to de-exclude `Server`, adding tests as necessary.